ofrak_components/ofrak_components/uf2.py

```diff
@@ -66,10 +65,6 @@ class Uf2BlockHeader(ResourceAttributes):
     num_blocks: int
     filesize_family_id: int
 
-    @index
-    def BlockNo(self) -> int:
-        return self.block_no
```

Comment:
Why remove this instead of adding a test for it? Are you confident people won't want to sort by block number in their results?

Reply:
This index isn't needed for current functionality of Uf2 unpacker and packer.

If a future use case required it, it can always be added back.